compass/ocean/tests/dam_break/forward.py

```diff
@@ -50,18 +54,37 @@ def __init__(self, test_case, resolution, name='forward', subdir=None,
 
         self.add_namelist_file('compass.ocean.tests.dam_break',
                                'namelist.forward')
-        self.add_streams_file('compass.ocean.tests.dam_break',
-                              'streams.forward')
-
-        input_path = '../initial_state'
-        self.add_input_file(filename='mesh.nc',
-                            target=f'{input_path}/culled_mesh.nc')
-
-        self.add_input_file(filename='init.nc',
-                            target=f'{input_path}/ocean.nc')
 
-        self.add_input_file(filename='graph.info',
-                            target=f'{input_path}/culled_graph.info')
+        if use_lts:
+            self.add_namelist_options(
+                {'config_time_integrator': "'LTS'"})
+            self.add_namelist_options(
+                {'config_dt_scaling_LTS': "4"})
+            self.add_namelist_options(
+                {'config_number_of_time_levels': "4"})
```

Comment:
```suggestion
            self.add_namelist_options(
                {'config_time_integrator': "'LTS'",
                'config_dt_scaling_LTS': "4",
                'config_number_of_time_levels': "4"})
```
@gcapodag It would just look like this

Reply:
Thanks!